src/barnetilsyn/BarnetilsynApp.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import Feilside from '../components/feil/Feilside';
+import hentToggles from '../toggles/api';
+import NavFrontendSpinner from 'nav-frontend-spinner';
+import Søknadsdialog from './Søknadsdialog';
+import TestsideInformasjon from '../components/TestsideInformasjon';
+import { hentPersonData } from '../utils/søknad';
+import { PersonActionTypes, usePersonContext } from '../context/PersonContext';
+import { Switch, Route } from 'react-router-dom';
+import { ToggleName } from '../models/toggles';
+import {
+  autentiseringsInterceptor,
+  verifiserAtBrukerErAutentisert,
+} from '../utils/autentisering';
+import mockPersonMedBarn from '../mock/mockPerson.json';
+import mockPersonUtenBarn from '../mock/mockPersonUtenBarn.json';
+import mockToggles from '../mock/mockToggles.json';
+import { settLabelOgVerdi } from '../utils/søknad';
+import { standardLabelsBarn } from '../helpers/labels';
+import { useBarnetilsynSøknad } from './BarnetilsynContext';
+import { useToggles } from '../context/TogglesContext';
+import { IPerson } from '../models/person';
+import { Helmet } from 'react-helmet';
+import { erLokaltMedMock } from '../utils/miljø';
+
+const BarnetilsynApp = () => {
+  const [autentisert, settAutentisering] = useState<boolean>(false);
+  const [fetching, settFetching] = useState<boolean>(true);
+  const [error, settError] = useState<boolean>(false);
+  const { settPerson } = usePersonContext();
+  const {
+    søknad,
+    settSøknad,
+    hentMellomlagretOvergangsstønad,
```

Comment:
Kan nok ikke bruke `...mellomlagretOvergangsstønad` for barnetilsyn - da må vi legge inn nytt endepunkt i dokument og endre bruken for barnetilsyn.